### ISSUE TITLE
slim example not work for mobilenet_v2, request an explicit example

### ISSUE THREAD

**Issue by ywang370** [tensorflow/models#4520]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:  slim
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:  source
- **TensorFlow version (use command below)**: 1.80
- **Bazel version (if compiling from source)**: N.A
- **CUDA/cuDNN version**:  Cuda 9.0
- **GPU model and memory**:  1080 Ti
- **Exact command to reproduce**:   
https://github.com/tensorflow/models/blob/master/research/slim/slim_walkthrough.ipynb

You can collect some of this information using our environment capture script:  

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

I go through the ipython notebook provided in slim folder, it works well for me. However, when I try to replace the model with mobile net,  I find that it is not working for me.   Therefore,  an example for mobile net would be great.
 
### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

Example of fine tune  mobile net on flower dataset.  I follow the code from fine tune example in:
https://github.com/tensorflow/models/blob/master/research/slim/slim_walkthrough.ipynb

```import os
from datasets import flowers
from nets import inception
from preprocessing import vgg_preprocessing
from tensorflow.contrib import slim

def  get_init_fn():

     checkpoint_exclude_scopes=["MobilenetV2/Logits, MobilenetV2/Predictions, MobilenetV2/predics"]
     exclusions = [scope.strip() for scope in checkpoint_exclude_scopes]
     variables_to_restore = []
     for var in slim.get_model_variables():
         for exclusion in exclusions:
              if var.op.name.startswith(exclusion):
                 break
             else:
                variables_to_restore.append(var)

     return slim.assign_from_checkpoint_fn(
      os.path.join(checkpoints_dir, 'mobilenet_v2_1.0_224.ckpt'),
      variables_to_restore)

 train_dir = '/tmp/inception_finetuned/'

 with tf.Graph().as_default():
    tf.logging.set_verbosity(tf.logging.INFO)
    
    dataset = flowers.get_split('train', flowers_data_dir)
    images, _, labels = load_batch(dataset, height=image_size, width=image_size)
    
    # Create the model, use the default arg scope to configure the batch norm parameters.
    with slim.arg_scope(mobilenet_v2.training_scope(is_training=True)):
         logits, _ = mobilenet_v2.mobilenet(images, num_classes=dataset.num_classes)

    # Specify the loss function:
    one_hot_labels = slim.one_hot_encoding(labels, dataset.num_classes)
    slim.losses.softmax_cross_entropy(logits, one_hot_labels)
    total_loss = slim.losses.get_total_loss()

    # Create some summaries to visualize the training process:
    tf.summary.scalar('losses/Total Loss', total_loss)
  
    # Specify the optimizer and create the train op:
    optimizer = tf.train.AdamOptimizer(learning_rate=0.01)
    train_op = slim.learning.create_train_op(total_loss, optimizer)
    
    # Run the training:
    final_loss = slim.learning.train(
        train_op,
        logdir=train_dir,
        init_fn=get_init_fn(),
        number_of_steps=2)
        
  
  print('Finished training. Last batch loss %f' % final_loss)

With code you can see, I only replace the inception v3 to mobileNet.  

Error from following shows that the last layer is not match.  I guess I already exclude the logits layer in mobile net. 

InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [5] rhs shape= [1001]
         [[Node: save/Assign_10 = Assign[T=DT_FLOAT, _class=["loc:@MobilenetV2/Logits/Conv2d_1c_1x1/biases"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](MobilenetV2/Logits/Conv2d_1c_1x1/biases,save/RestoreV2/_1)]]
         [[Node: save/RestoreV2/_78 = _Send[T=DT_FLOAT, client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_84_save/RestoreV2", _device="/job:localhost/replica:0/task:0/device:CPU:0"](save/RestoreV2:107)]]

**Comment by ywang370** [comment]
exclude scope needs split the string.
